test/Rollup.spec.ts

```diff
@@ -257,11 +257,11 @@ contract("Rollup", async function (accounts) {
       from: AliceAccountMP,
       to: BobAccountMP,
     };
-
     // process transaction validity with process tx
     var result = await rollupCoreInstance.processTx(
       currentRoot,
       accountRoot,
+      tx.signature,
       await utils.TxToBytes(tx),
```

Comment:
I know you might have a reason to do the current way, but don't we just add the signature to `BytesFromTxDeconstructed`?

Reply:
Ah! No the tx bytes dont contain the signature: https://github.com/thehubbleproject/contracts/blob/master/contracts/libs/RollupUtils.sol#L61

Only the compressedTx contains the signature. 

The way we see Tx is Tx{Data, Signature, Type, ChainID} so encodedTx will not have the signature.

Reply:
Got it.